tests/antithesis/config/manifests/default-etcd-3-replicas.yaml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
I think we just need this single file.

Reply:
Maybe this is not clear, but this file is generated from `helm template charts/etcd > default-etcd-3-replicas.yaml`. I haven't included that in the make file yet.  With the helm chart templates, we can use it to generate a different manifest by supplying different values.  This is now used for making sure that the replica count will result in the correct bootstrap value for each instance too.

btw, Helm is only for templating and not used for the actual deployment.